stable/kubernetes-dashboard/templates/deployment.yaml

```diff
@@ -30,6 +30,10 @@ spec:
       - name: {{ .Chart.Name }}
         image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
         imagePullPolicy: {{ .Values.image.pullPolicy }}
+        args:
+        {{- range $key, $value := .Values.extraArgs }}
+          - --{{ $key }}={{ $value }}
+        {{- end }}
```

Comment:
Can you wrap `args` in an if statement to only include the section if there are args.